src/Microsoft.CSharp/src/Microsoft/CSharp/RuntimeBinder/Semantics/Operators.cs

```diff
@@ -1722,7 +1722,7 @@ private Expr BindIncOp(ExpressionKind ek, EXPRFLAG flags, Expr arg, UnaOpFullSig
             FUNDTYPE ft = typeRaw.fundType();
             if (ft == FUNDTYPE.FT_R8 || ft == FUNDTYPE.FT_R4)
             {
-                flags = ~EXPRFLAG.EXF_CHECKOVERFLOW;
+                flags &= ~EXPRFLAG.EXF_CHECKOVERFLOW;
```

Comment:
This and the similar below fixes a clear mistake in an attempt to unset that bit, that is probably harmless if you aren't doing a test-run with extra assertions that flags doesn't have the base-call bit set, but should be fixed all the same.